src/algebra/module/submodule_lattice.lean

```diff
@@ -106,6 +106,10 @@ variables (R)
 @[simp] lemma restrict_scalars_top : restrict_scalars S (⊤ : submodule R M) = ⊤ := rfl
 end
 
+@[simp] lemma restrict_scalars_eq_top_iff {p : submodule R M} :
+  restrict_scalars R p = ⊤ ↔ p = ⊤ :=
```

Comment:
I assume you meant:
```suggestion
  restrict_scalars S p = ⊤ ↔ p = ⊤ :=
```

Reply:
Indeed, a copy-paste error, well spotted!